modules/installing-rhv-setting-up-ca-certificate.adoc

```diff
@@ -16,16 +16,16 @@ Later, you provide the certificate to the installation program.
 .Procedure
 
 . Use either of these two methods to download the CA certificate:
-** Go to the Manager's webpage, `\https://<engine-fqdn>/ovirt-engine/`. Then, under *Downloads*, click the *CA Certificate* link.
+** Go to the {rh-virtualization-engine-name}'s webpage, `\https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/`. Then, under *Downloads*, click the *CA Certificate* link.
```

Comment:
Does this module need to appear in the default install and install with customizations? I am not seeing it. I see it in the restricted install

Reply:
No, this should not appear there.